### ISSUE TITLE
Possible to detect audio play?

### ISSUE THREAD

**Issue by ApoorvSaxena** [ChromeDevTools/devtools-protocol#69]
Is it possible to detect audio play via CDP, e.g. when video is playing on youtube we are able to see speaker icon on the tab. Using this API (if already available) one should be notified via callback, when an audio is played on a webpage.

Also, video auto play will be blocked by default, will it be available as an API to observe as well when a video gets autoplayed.

**Comment by JoelEinbinder** [comment]
Looks like a duplicate of https://github.com/GoogleChrome/puppeteer/issues/516

Seems doable.

**Comment by TimvdLippe** [comment]
This repository is related to Chrome DevTools Protocol, but does not track issues regarding its definition or implementation. If you want to file an issue for the Chrome DevTools Protocol, please open an issue on https://crbug.com under [`component: Platform>DevTools>Platform`](https://bugs.chromium.org/p/chromium/issues/entry?components=Platform%3EDevTools%3EPlatform). Thanks in advance!
